validation_info.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
@alan-baker This file is needed in case the file locations would ever change it would be a pain to have to track various locations to look. Debated about putting file in `utils` folder, but figured if put at root there will never be a guess where this file is (which is important)

Reply:
It seems like we know which VUIDs are checked by looking solely at validation_state.cpp. I'm not clear on the use of this file.

Reply:
If for whatever reason that location changes, then the Validation Layers have to keep a list of every location that it might have been

Reply:
It seems like the name of the file should be `vulkan_validation_info.json` at least. Also, I'm not clear on why the test files are listed. What is the validation layer script doing exactly?

Reply:
1. I don't know the plan for WebGPU/CL/futureApis are and if it would care for something similar, so that's what I have the `target : Vulkan` section... happy to rename the file, but then in the future there should be a `api_validation_info.json` if someone else does need one

2.  [Here is the current code](https://github.com/KhronosGroup/Vulkan-ValidationLayers/pull/2145/files#diff-d648cc0d8fa02012cc79869be6d282e5R762). When you run the Validation Layer stats script it looks from non-commented strings in the code and then builds this [database](https://vulkan.lunarg.com/doc/sdk/1.2.148.1/windows/validation_error_database.html) of info of what is done and what isn't. The reason for the test is it nice to know if there is an actual test testing the validation

So sorry in making you become a Vulkan validation expert with this PR, but hopefully this answers your questions

Reply:
Wouldn't it be better to add this file in the event we move the strings in the future?

Reply:
I think Dan meant, wait to add this file until it is required and I tend to agree with that.

Reply:
I assume that the 2 paths I have listed in the file currently won't change unless there is a large refactor of the repo, but if that event occurs, feel adding a file then doesn't do much as there is always this "need to check the range from first adding VUs to the file" check that will always occur

Regardless, thanks for weighing in, will remove file, hopefully everything is else is good to go otherwise